types/block.go

```diff
@@ -41,21 +43,13 @@ type Commit struct {
 	Signatures []Signature // most of the time this is a single signature
 }
 
-type Validator struct {
-	PublicKey []byte
-}
-
-type ValidatorSet struct {
-	Validators []Validator
-}
-
 // SignedHeader combines Header and its Commit.
 //
 // Used mostly for gossiping.
 type SignedHeader struct {
 	Header
 	Commit     Commit
-	Validators ValidatorSet
+	Validators *types.ValidatorSet
```

Comment:
suggestion: we could change `types` to `tmtypes` to maintain consistency with other files where we use `tmtypes`?